rnd/infra/terraform/environments/dev.tfvars

```diff
@@ -0,0 +1,11 @@
+project_id     = "agpt-dev"
```

Comment:
This is fine how it is, however my preference is to use white space to give a visual grouping of what variables are logically grouped together. Such as general, networking or cluster ones

Reply:
yeah fair call - as we end up with more modules, I was gonna re do how we do environments.. I think that will help the visual readability. but if you prefer, I can also just add the white space now lmk